fladgejt/webui/hodnotenia.py

```diff
@@ -53,3 +53,47 @@ def get_priemery(self, zapisny_list_key):
                           datum_vypoctu=row['priemerInfoDatum'])
                   for row in app.d.priemeryTable.all_rows()]
         return [result, None]
+
+    def get_priebezne_hodnotenia(self, zapisny_list_key):
+        try:
+            app = self._open_priebezne_hodnotenie_app(zapisny_list_key)
+        except CantOpenApplication:
+            return [[], "Priebežné hodnotenia pre tento zápisný list nie sú dostupné."]
+
+        # Ak uz bola aplikacia otvorena pred tymto volanim, stlacime "Refresh".
+        if getattr(app, 'freshly_opened', True):
+            app.freshly_opened = False
+        else:
+            app.d.refreshButton.click()
+
+        studium_key, akademicky_rok = decode_key(zapisny_list_key)
+
+        result = []
+        for i, row in enumerate(app.d.predmetySPriebeznymHodnotenimTable.all_rows()):
+            # Klikneme na predmet s priebeznym hodnotenim a pockame na nacitanie
+            with app.collect_operations() as ops:
+                app.d.predmetySPriebeznymHodnotenimTable.select(i)
+            if ops:
+                app.awaited_refresh_dialog(ops)
+
+            # Klikneme na button nacitatHodnotenie
+            with app.collect_operations() as ops:
+                app.d.nacitatHodnotenieButton.click()
+            if ops:
```

Comment:
Ked kliknem na nacitatHodnotenieButton, tak sa nachvilku zobrazi taky box ze "Cakajte prosim".
Myslel som, ze kvoli tomu tam je ten wait.
Skusal som a ide to aj bez `with` a `if`, takze to zmazem.